data/macros.meson

```diff
@@ -45,3 +45,8 @@
         --num-processes %{_smp_build_ncpus} \
         --print-errorlogs \
         %{nil}}
+
+%meson_buildrequires \
+     %{shrink: mesongenbuildreq %{__meson} \
+    %{nil}}
+    
```

Comment:
This file is maintained by @ignatenkobrain.
@DaanDeMeyer @keszybz have demonstrated interest as stakeholders in the past.

Please review (since none of the meson core maintainers know the first thing about rpm, as far as I know :D)